<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>481</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>941</width>
    <height>481</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>941</width>
    <height>481</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>6</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Logout</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(55, 64, 106);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_pesquisa">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 69, 0);
color: rgb(255, 255, 255);
border-radius: 15px</string>
    </property>
    <property name="text">
     <string>PESQUISAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cadastro">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 69, 0);
color: rgb(255, 255, 255);
border-radius: 15px</string>
    </property>
    <property name="text">
     <string>CADASTRAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_logout">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 69, 0);
color: rgb(255, 255, 255);
border-radius: 15px</string>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-230</x>
      <y>-120</y>
      <width>761</width>
      <height>740</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 375px 370px;
background-color: rgb(219, 222, 238);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>42</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 222, 238);
color: rgb(55, 64, 106);</string>
    </property>
    <property name="text">
     <string>SEJAM BEM-VINDOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(219, 222, 238);
color: rgb(55, 64, 106);</string>
    </property>
    <property name="text">
     <string>Escolha uma opção     ao lado para prosseguir:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>941</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 95, 30);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>                                                                                                                                                                                                                                                                          Criado por: Big Brain Team</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton_pesquisa</zorder>
   <zorder>pushButton_cadastro</zorder>
   <zorder>pushButton_logout</zorder>
   <zorder>label_3</zorder>
   <zorder>label_2</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
